### PR TITLE
i added the files of linuxcommads git commands and scrum.org

### PR DESCRIPTION
**Pull request by prudhvidhargudimalla**
these are the files

### CODE REVIEW

**Review by kulkarni90 — COMMENTED** [review]
command files are though short it fulfilled the task 

scrum file has only about sprint ..please  add about scrum.

**Comment by pradeepm2406** [comment]
1. please give introduction about git and then write about commands.
2. there are only 8 commands mentioned in linux file. Please add 2 more commands
3. In scrum file, more points need to be added

File: prudhvi1/git@commands.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+1)git pull request = a method of submitting contributions to a repository
+2)fork = copy of a repository owned by a different user
+3)Head = represents your current working directory
+4)clone= local version of a repository, including all commits and branches
+5)git branch -d branch name = deletes the specific branch
+6)git fetch = downloads all history from the remote tracking branches
+7)git reset = undoes all commits after [commit], preserving changes locally
+8)git add = snapshots the file in preparation for versioning
+9)git mv [existing -path] [new-path] = change an existing file path and stage the move
+10)git rm [file] = delete the file from project and stage the removal for commit
```
Comment (david1412) [inline prudhvi1/git@commands.txt]: Good work!
Please add what is Git? some intro missing?

File: prudhvi1/linux@commands.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+linux commands 
+1)pwd = It prints the path of the working directory, starting from the roots
+2)echo =Used to display line of text/string that are passed as an argument
+3)cd  = change directoty
+4)touch= Used to create, change and modify timestamps of a file
+5)mv=Used to move one or more files or directories from one place to another in file system like UNIX
+6)escp+:1,$d=deleting all the lines from the file
+7)dd= deleting line to line
+8)dD= deleting the paragraph
+
```
Comment (david1412) [inline prudhvi1/linux@commands.txt]: 2 more commands are missing.
Remove the last 2 lines, No spaces needed.

**Review by pradeepm2406 — COMMENTED** [review]
1. please give introduction about git and then write about commands.
2. there are only 8 commands mentioned in linux file. Please add 2 more commands
3. In scrum file, more points need to be added

**Review by david1412 — COMMENTED** [review]
See the comments.
Use semantic formatting to write the PR name

File: prudhvi1/scrum@txt [hunk 3]
```diff
@@ -1,4 +1,14 @@
...
+   Scrum is:
+   1)Lightweight
+   2)Simple to understand
+   3)Difficult to master
+                    what is sprint
 Sprint is one timeboxed iteration of a continuous development cycle. Within a Sprint, planned amount of work has to be completed by the team and made ready for review. The term is mainly used in Scrum Agile methodology but somewhat basic idea of Kanban continuous delivery is also essence of Sprint Scrum.
 Sprint literal meaning is a short race at full speed. Accordingly, teams usually define a short duration of a Sprint up to 2-4 weeks. Team collaboratively sets their target with Product Owner as “Sprint Goal” and plan their work in “Sprint backlog”. As soon race starts after planning session, team work together to complete planned work effectively and make it ready for review by the end of that period.
 High level User Stories readiness in Product backlog is the prerequisite of starting a Sprint Cycle. Sprint Analytics help Scrum Master and Product Owner to know the progress of Sprint in a glance. It is the place to define Sprint Goal and Definition of Done for each Sprint.
```
Comment (david1412) [inline prudhvi1/scrum@txt:14]: More explanation is needed and list 3-5 artefacts

**Review by david1412 — COMMENTED** [review]
Do changes in the scrum.text, remaining files look good.

**Review by nithindeva — APPROVED** [review]

**Pull request merged**